packages/client/src/__tests__/integration/happy/interactive-transactions-postgres/test.ts

```diff
@@ -192,15 +192,15 @@ describe('interactive transactions', () => {
 
     await expect(result).rejects.toThrowErrorMatchingInlineSnapshot(`
 
-                                    Invalid \`prisma.user.create()\` invocation in
-                                    /client/src/__tests__/integration/happy/interactive-transactions-postgres/test.ts:0:0
+                                                Invalid \`prisma.user.create()\` invocation in
```

Comment:
I am not quite sure why those snapshots were updated, tbh. Should it be like that?

Reply:
Long standing problem with the snapshots generation: https://github.com/prisma/prisma/issues/8031

We should probably rename and expand that.

Reply:
It's a known issue, yes. You may revert unrelated changes if you want to minimize churn, or you may leave it as is, either way is fine.